.github/actions/k8s-compatibility-test/action.yaml

```
@@ -16,7 +16,7 @@ runs:
         cluster-name: "test-cluster-1"
         args: >-
           --agents 2
-          --image docker.io/rancher/k3s:v1.27.3-k3s1
+          --image docker.io/rancher/k3s:v1.28.2+k3s1
```

I think the tag is incorrect
```suggestion
          --image docker.io/rancher/k3s:v1.28.2-k3s1
```